### PR TITLE
Fix editing participant

### PR DESCRIPTION
**Pull request by Kajol-Kumari**
Fix: #2624 

Editing participant data adds data to other fields fixed.

### CODE REVIEW

File: frontend/src/js/controllers/dashCtrl.js [hunk 1]
```diff
@@ -131,7 +131,7 @@
                 var status = response.status;
                 var details = response.data;
                 if (status == 200) {
-                    vm.participatedTeamCount = details.count;
+                    vm.participatedTeamCount = details.results.length;
```
Comment (Suryansh5545) [inline frontend/src/js/controllers/dashCtrl.js]: Why these change?

File: frontend/src/js/controllers/dashCtrl.js [hunk 2]
```diff
@@ -99,7 +99,7 @@
                 var status = response.status;
                 var details = response.data;
                 if (status == 200) {
-                    vm.hostTeamCount = details.count;
+                    vm.hostTeamCount = details.results.length;
```
Comment (Suryansh5545) [inline frontend/src/js/controllers/dashCtrl.js]: same here

File: frontend/tests/controllers-test/dashCtrl.test.js [hunk 3]
```diff
@@ -167,22 +167,24 @@ describe('Unit tests for dashboard controller', function () {
...
+                        team_url: "https://team1.url"
+                    },
+                    {
+                        id: 2,
+                        team_name: "Host team name 2",
+                        created_by: "user 2",
+                        team_url: "https://team2.url"
+                    },
+                ]
+        };
```
Comment (Suryansh5545) [inline frontend/tests/controllers-test/dashCtrl.test.js]: undo these changes

File: frontend/tests/controllers-test/dashCtrl.test.js [hunk 4]
```diff
@@ -220,22 +222,24 @@ describe('Unit tests for dashboard controller', function () {
...
+                        team_url: "https://team1.url"
+                    },
+                    {
+                        id: 2,
+                        team_name: "Participants team name 2",
+                        created_by: "user 2",
+                        team_url: "https://team2.url"
+                    },
+                ]
+            };
```
Comment (Suryansh5545) [inline frontend/tests/controllers-test/dashCtrl.test.js]: same here

**Comment by Suryansh5545** [comment]
@Kajol-Kumari I am unable to create the participant team. Please fix it 
Selection_053

**Comment by Kajol-Kumari** [comment]
Done @Suryansh5545

File: frontend/tests/controllers-test/dashCtrl.test.js [hunk 5]
```diff
@@ -168,19 +168,19 @@ describe('Unit tests for dashboard controller', function () {
...
+                        created_by: "user 1",
+                        team_url: "https://team1.url"
+                    },
+                    {
+                        id: 2,
+                        team_name: "Host team name 2",
+                        created_by: "user 2",
+                        team_url: "https://team2.url"
+                    },
+                ];
```
Comment (Suryansh5545) [inline frontend/tests/controllers-test/dashCtrl.test.js]: Remove these changes also

File: frontend/tests/controllers-test/dashCtrl.test.js [hunk 6]
```diff
@@ -221,19 +221,19 @@ describe('Unit tests for dashboard controller', function () {
...
+                        created_by: "user 1",
+                        team_url: "https://team1.url"
+                    },
+                    {
+                        id: 2,
+                        team_name: "Participants team name 2",
+                        created_by: "user 2",
+                        team_url: "https://team2.url"
+                    },
+                ]
```
Comment (Suryansh5545) [inline frontend/tests/controllers-test/dashCtrl.test.js]: same here

**Comment by Suryansh5545** [comment]
@Kajol-Kumari it is working but can we use a more understandable name instead of pname or purl?

**Comment by Suryansh5545** [comment]
@Kajol-Kumari LGTM:+1:

**Comment by Suryansh5545** [comment]
@Ram81 @RishabhJain2018 Please take a look

**Comment by Kajol-Kumari** [comment]
Done!

File: frontend/src/js/controllers/teamsCtrl.js [hunk 7]
```diff
@@ -25,6 +25,7 @@
         vm.isPrev = '';
         vm.team.error = false;
         vm.showPagination = false;
+        vm.participantTeamId = null;
```
Comment (Sanji515) [inline frontend/src/js/controllers/teamsCtrl.js]: Do we need this change? @Kajol-Kumari

File: frontend/src/js/controllers/teamsCtrl.js [hunk 8]
```diff
@@ -370,8 +371,8 @@
             parameters.token = userKey;
             parameters.callback = {
                 onSuccess: function(response) {
-                    vm.team.name = response.data.team_name;
-                    vm.team.url = response.data.team_url;
+                    vm.team.participantName = response.data.team_name;
```
Comment (Sanji515) [inline frontend/src/js/controllers/teamsCtrl.js]: can we please change the name `participantName` --> `teamName`

File: frontend/src/js/controllers/teamsCtrl.js [hunk 9]
```diff
@@ -370,8 +371,8 @@
...
+                    vm.team.participantNameUrl = response.data.team_url;
```
Comment (Sanji515) [inline frontend/src/js/controllers/teamsCtrl.js]: same here `participantNameUrl` --> `teamUrl`

File: frontend/src/js/controllers/teamsCtrl.js (hunk 7 above)
Comment (Kajol-Kumari) [reply frontend/src/js/controllers/teamsCtrl.js]: > Do we need this change? @Kajol-Kumari

@Sanji515 This line is actually not needed but i used it just to denote that currently participantTeamId does not have any value but it will have later on. But I will remove this line as it is not affecting any functionality.

**Comment by Kajol-Kumari** [comment]
@Sanji515 I am closing this pull request and opening a new one with all your suggested changes.I had removed this particular branch which i will make sure not to do till the pull request gets merged.

**Comment by Sanji515** [comment]
> @Sanji515 I am closing this pull request and opening a new one with all your suggested changes.I had removed this particular branch which i will make sure not to do till the pull request gets merged.

Hey @Kajol-Kumari Even if you have deleted your local branch still you can pull your PR `git fetch origin pull/ID/head:BRANCHNAME`. Please take a look [here](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/checking-out-pull-requests-locally) :)

**Comment by Kajol-Kumari** [comment]
> > @Sanji515 I am closing this pull request and opening a new one with all your suggested changes.I had removed this particular branch which i will make sure not to do till the pull request gets merged.
> 
> Hey @Kajol-Kumari Even if you have deleted your local branch still you can pull your PR `git fetch origin pull/ID/head:BRANCHNAME`. Please take a look [here](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/checking-out-pull-requests-locally) :)

Thanks for this document. This will help me a lot.
